### DISCUSSION TITLE
google-zmusi-cie-na-przejscie-platny-google-wokspace-migracja/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#101] [General]
# google-zmusi-cie-na-przejscie-platny-google-wokspace-migracja/

Wolny świat, wolne myśly, wolne słowa.

https://dariusz.wieckiewicz.org/google-zmusi-cie-na-przejscie-platny-google-wokspace-migracja/

**Comment by xpil** [comment]
"płacenie £9.20 za użytkownika" - ja to wiem i ty to wiesz, ale może jakiś czytelnik tego nie wie, że chodzi o £9.20 *miesięcznie*. Warto przy cenach subskrypcji podawać również okresy.

Co do meritum, też mnie wkurzają z tymi planami. Wypatruję sensownych alternatyw, ale póki co słabo to wygląda...

**Reply by idarek** [reply]
Dzięki za zwrócenie uwagi, już się poprawia.

Ja będę przenosił cię na standardowe konto Google. Napisze szczegółowo jak mi poszło :) będzie ciekawie...
